anchore_engine/services/policy_engine/api/controllers/synchronous_operations.py

```diff
@@ -995,11 +994,11 @@ def get_image_vulnerabilities(user_id, image_id, force_refresh=False, vendor_onl
                     link = vulnerability_cpe.parent.link
                     if not link:
                         link = "{}/query/vulnerabilities?id={}".format(
-                            api_endpoint, vulnerability_cpe.vulnerability_id
+                            api_endpoint, vulnerability_cpe.parent.normalized_id
```

Comment:
Rollback this change. This is related to the vulnerability query api which will be based off the regular ID, not the normalized ID